### DISCUSSION TITLE
How to disable a icon preview on VSCode

### DISCUSSION THREAD

**Discussion by aliuq** [unocss/unocss#3725] [Q&A] [answered]
![image](https://github.com/unocss/unocss/assets/11707934/17c50569-f106-41dc-ad4b-5dd12107cb77)

**Comment by telangY** [comment]
I have encountered it before, but I forgot how to solve it. You can try restarting it

**Reply by aliuq** [reply]
Hi, Can you still remember it? 🤣

**Reply by telangY** [reply]
You can check if element ui plugin is installed in vscode, and if so, please del it.

**Comment by aliuq** [comment, marked as answer]
https://github.com/antfu/vscode-iconify/issues/41
